RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -818,20 +818,20 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       }
       alpaka::syncBlockThreads(acc);
 
-      // Occupancy matrix for 0.8 GeV pT Cut
+      // Occupancy matrix for 0.8 GeV pT Cut, 99.99%
       constexpr int p08_occupancy_matrix[4][4] = {
-          {543, 235, 88, 46},  // category 0
-          {755, 347, 0, 0},    // category 1
-          {0, 0, 0, 0},        // category 2
-          {0, 38, 46, 39}      // category 3
+          {1373, 702, 326, 83},  // category 0
+          {1323, 653, 0, 0},     // category 1
```

Comment:
close to x3 increase; similar to T5s: was it a different quantile previously or is this a result of an increase in the LS occupancy?

Reply:
I updated the percentile here from 99.9% to 99.99% to match the other objects.